core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -527,7 +536,43 @@ public boolean isTerminated() {
     return terminated;
   }
 
-  /*
+  @Override
+  public ConnectivityState getState(boolean requestConnection) {
+    // TODO: handle requestConnection. At this point, not really sure how.
```

Comment:
I've moved discussion to [the issue](https://github.com/grpc/grpc-java/issues/2292#issuecomment-284480422), this comment will get buried after code changes.